scripts/bundleSize/__mocks__/pageTypeBundleExtractor.js

```diff
@@ -1,158 +1,158 @@
+      ];
+    }
+    if (pageType === 'MediaAssetPage') {
+      return [
+        'modern.MediaAssetPage-88a3c260.b7ec8c9c.js',
+        'modern.commons-1111.js',
+        'modern.commons-2222.js',
+        'modern.1111-lib-1111.js',
+        'modern.3333-lib-2222.js',
+        'modern.shared-1111.js',
+      ];
+    }
+    if (pageType === 'MostReadPage') {
+      return [
+        'modern.MostReadPage-31ecd969.7484ff05.js',
+        'modern.commons-1111.js',
+        'modern.commons-2222.js',
+        'modern.1111-lib-1111.js',
+        'modern.shared-1111.js',
+        'modern.shared-2222.js',
+        'modern.shared-3333.js',
+      ];
+    }
+    if (pageType === 'MostWatchedPage') {
+      return [
+        'modern.MostWatchedPage-31ecd969.7484ff05.js',
+        'modern.commons-1111.js',
+        'modern.commons-2222.js',
+        'modern.1111-lib-1111.js',
+        'modern.shared-1111.js',
+        'modern.shared-2222.js',
+        'modern.shared-3333.js',
+      ];
+    }
+    if (pageType === 'OnDemandAudioPage') {
+      return [
+        'modern.OnDemandAudioPage-31ecd969.ec6af2d0.js',
+        'modern.commons-1111.js',
+        'modern.commons-2222.js',
+        'modern.1111-lib-1111.js',
+        'modern.shared-2222.js',
+        'modern.shared-3333.js',
+      ];
+    }
+    if (pageType === 'OnDemandTvPage') {
+      return [
+        'modern.OnDemandTvPage-31ecd969.de41ab7f.js',
+        'modern.commons-1111.js',
+        'modern.1111-lib-1111.js',
+        'modern.3333-lib-2222.js',
+        'modern.shared-1111.js',
+      ];
+    }
+    if (pageType === 'PhotoGalleryPage') {
+      return [
+        'modern.PhotoGalleryPage-e94df663.a733283a.js',
+        'modern.commons-1111.js',
+        'modern.commons-2222.js',
+        'modern.1111-lib-1111.js',
+        'modern.shared-1111.js',
+        'modern.shared-3333.js',
+      ];
+    }
+    if (pageType === 'ErrorPage') {
+      return [
+        'modern.ErrorPage-31ecd969.31473c35.js',
+        'modern.commons-1111.js',
+        'modern.commons-2222.js',
+        'modern.1111-lib-1111.js',
+        'modern.shared-1111.js',
+        'modern.shared-2222.js',
+        'modern.shared-3333.js',
+      ];
+    }
+    if (pageType === 'IdxPage') {
+      return [
+        'modern.IdxPage-31ecd969.31473c35.js',
+        'modern.commons-1111.js',
+        'modern.commons-2222.js',
+        'modern.1111-lib-1111.js',
+        'modern.3333-lib-2222.js',
+        'modern.shared-1111.js',
+      ];
+    }
+    if (pageType === 'FeatureIdxPage') {
+      return [
+        'modern.FeatureIdxPage-31ecd969.31473c35.js',
+        'modern.commons-1111.js',
+        'modern.commons-2222.js',
+        'modern.1111-lib-1111.js',
+        'modern.3333-lib-2222.js',
+        'modern.shared-1111.js',
+      ];
+    }
 
-  throw Error(`page type ${pageType} not mocked`);
-};
-
-exports.extractBundlesForPageType = extractBundlesForPageType;
+    throw Error(`page type ${pageType} not mocked`);
+  },
+}
```

Comment:
Reformat mock module to work with `unstable_mockModule`: https://github.com/bbc/simorgh/pull/10937/files#diff-186540c5991ff0121d0c796909193883e3289c4b84c8d327af446173a6423ecfR5 

Mock modules must always have a `default` export to work with `unstable_mockModule`